packages/ovm-toolchain/src/buidler-plugins/buidler-ovm-node.ts

```diff
@@ -20,6 +20,7 @@ extendEnvironment(async (bre) => {
       stateManager: vm.stateManager,
       emGasLimit: gasLimit,
     })
+    await ovm.init()
```

Comment:
Ah this is so that we don't do that init step during the Synthetix tests & break snapshotting -- is that right?

Reply:
I believe @kfichter mentioned this was to ensure the Execution manager state is correctly initialized, which was messing with timestamps?